### DISCUSSION TITLE
Last Data Refresh

### DISCUSSION THREAD

**Discussion by rcfoyn** [DaxStudio/DaxStudio#649] [Q&A] [answered]
In VertiPaq Analyzer Metrics Summary, please explain in detail what exactly does Last Data Refresh refer to. Does it change if I process only one table? Does it change on any mode of processing, i.e. Full, Default, Clear, Recalc?

Reference:
https://daxstudio.org/documentation/features/model-metrics/

**Comment by dgosbell** [comment, marked as answer]
This date comes from the model metadata. It is the last time any type of data refresh was performed.

> Does it change if I process only one table? 

Yes

> Does it change on any mode of processing, i.e. Full, Default, Clear, Recalc?

Yes. In fact Power BI Desktop does a process recalc when you open a pbix file so if you are connected to PBI Desktop this date will never be earlier than when you opened the file.
